src/04-invoicing/ledgers.apib

```diff
@@ -0,0 +1,17 @@
+### ledgers.list [GET /ledgers.list]
+
+Get a list of ledger accounts.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + department_id: `080aac72-ff1a-4627-bfe3-146b6eee979c` (string, optional) - The ID of the department you wish to filter on.
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `2aa4a6a9-9ce8-4851-a9b3-26aea2ea14c4` (string)
+                + number: 123 (number)
```

Comment:
It would also make sense to add the department_id here

Reply:
I'm not sure we really know the departmentId ?

Reply:
it is a column in the db inv_accounts, so we can add it and filter by it

Reply:
Oh, if it's already cleanly separated by department, yes, by all means, let's add it!